src/lang/modifyAst/addSweep.ts

```diff
@@ -136,14 +136,10 @@ export function addSweep({
     return sketchesExprList
   }
 
-  // Extra roundabout to find the trajectory (or path) declaration for the labeled argument
-  const pathNodePath = getNodePathFromSourceRange(
-    ast,
-    path.graphSelections[0].codeRef.range
-  )
+  // Find the path declaration for the labeled argument
   const pathDeclaration = getNodeFromPath<VariableDeclaration>(
     ast,
-    pathNodePath,
+    path.graphSelections[0].codeRef.pathToNode,
```

Comment:
@Irev-Dev @jtran I was playing with this as I remember you telling me to start trusting codeRef.pathToNode more. This change away from `getNodePathFromSourceRange` works perfectly for me when I'm creating a sweep, but not if I'm editing one. Just a data point, I can help investigate whenever

Related https://github.com/KittyCAD/modeling-app/pull/6737